test/tesselation.jl

```diff
@@ -45,5 +45,18 @@
     pts = randpoint3(10)
     pset = PointSet(pts)
     @test_throws AssertionError tesselate(pset, VoronoiTesselation(StableRNG(2024)))
+
+    # Test polygon order is the same as input points order
+    pts = randpoint2(10)
+    mesh = tesselate(pts, VoronoiTesselation(StableRNG(2024)))
+    @test all(zip(pts, mesh)) do (p, poly)
+      p ∈ poly && return true
+      # point is not in poly due to a floating point error.
+      # check if the target polygon's centroid is the closest
```

Comment:
```suggestion
      # check if the target polygon's centroid is the closest
```

Let's simplify the test to avoid these subtle floating point issues.